### ISSUE TITLE
[Issue]: channel forwarder not starting emulator on wii when installed on sd card

### ISSUE THREAD

**Issue by ivo71** [dborth/snes9xgx#1049]
### Describe the bug

After installing the channel forwarder, the channel doesn't start the emulator on the Wii.
I am using a 64GB sd card only, no usb. directory name of the emulator is left the same as in 4.5.3 zip file.

### To Reproduce

unzip emulator to sd card
unzip channel installer to sd card
go to homebrew channel
startup channel installer
install channel
go back to wii menu
start channel

### Expected behavior

channel starts emulator, but it doesn't the forwarder says loading but then goes back to the main wii menu.

**Comment by ivo71** [comment]
I used the files:
[Snes9xGX-4.5.3.zip](https://github.com/dborth/snes9xgx/releases/download/4.5.3/Snes9xGX-4.5.3.zip)
2.7 MB
Jun 16, 2022
[Snes9xGX-Channel-Installer.zip](https://github.com/dborth/snes9xgx/releases/download/4.5.3/Snes9xGX-Channel-Installer.zip)

**Comment by dborth** [comment]
Is the SD card formatted as fat32? Can you start the emulator from the homebrew channel? There really isn't much else to it and I really haven't seen anyone else complaining.

**Comment by ivo71** [comment]
Yes it is a FAT32 formatted 64GB SD Card. I tried it on two different wii consoles and on both it doesn't work. The forwarder just goes back to the main console screen. I can run the emulator from homebrew channel without any problems.
I tried also to unzip the emulator file to a usb harddrive and then the forwarder also didn't work.
I downloaded a different forwarder online called Snes9xgx_Animated_Forwarder_by_qwertymodo.wad and installed it with wiimodlite and it works fine. Then started your forwarder again and it just goes back to the main wii menu.
